lib/koans.ex

```diff
@@ -43,11 +43,11 @@ defmodule Koans do
     end
   end
   defmacro generate_test_method(name, number_of_args, body) do
-    answer_placeholders = create_vars(number_of_args)
-    multi_var = Blanks.replace(body, answer_placeholders)
+    answer_vars = for id <- 1..number_of_args, do: Macro.var(String.to_atom("answer#{id}"), Koans)
```

Comment:
Seems like you could have left the function that generates these values rather than doing it inline. That said, I don't feel strongly that it really improves the readability (it's all really complicated regardless).